README.md

```diff
@@ -97,7 +129,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer here
+this is used to parse the text as Url encoded data. Its used for the POST so we can create and update through JSON
```

Comment:
What does "URL encoded data" mean? Why do we need to use body parser in the first place?